components/core/src/clp_s/search/ast/SetDateLiteralPrecision.cpp

```
@@ -0,0 +1,34 @@
+        work_list.pop_back();
+
+        if (expr->has_only_expression_operands()) {
+            for (auto const& op : expr->get_op_list()) {
+                work_list.emplace_back(std::static_pointer_cast<Expression>(op));
+            }
+            continue;
+        }
+
+        for (auto const& op : expr->get_op_list()) {
+            if (auto date_literal{std::dynamic_pointer_cast<DateLiteral>(op)};
+                nullptr != date_literal)
```

Cppcheck (2.18.0)

[style] The function 'is_inverted' is never used.

(unusedFunction)